src/Form.php

```diff
@@ -258,8 +258,17 @@ public function onSubmit(\Closure $callback)
                 return $response;
             } catch (ValidationException $e) {
                 $response = [];
+                $openFirstSectionOnError = false;
+
                 foreach ($e->errors as $field => $error) {
                     $response[] = $this->error($field, $error);
+
+                    // If field inside Accordion section does not validate, open first AccordionSection
+                    if (!$openFirstSectionOnError && $this->getControl($field)->getOwner()->getOwner() instanceof \Atk4\Ui\AccordionSection) {
```

Comment:
this smells to me - `getOwner()->getOwner()` and below even triple times it seems it is single purpose code only and for more layout nesting it will not work

also, ideally it should not be handled in `Form`, as form should not care about it's childs too much

also Behat test should be added, so this code is covered - otherwise the logic might break soon - once you will satisfy all CI tests, you should see this in CodeCov report

Reply:
getOwner multiple times also is used in other view, like Crud. You are right, nested Accordions are then not supported - but since today it just won't work at all, this is a huge improvement to usability. And to me, nested Accordions in forms really is an edge case which would not be priority to me to support yet.